extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/HibernateValidatorProcessor.java

```diff
@@ -90,6 +91,8 @@
 
     private LocalesBuildTimeConfig localesBuildTimeConfig;
 
+    private HibernateValidatorBuildTimeConfig hibernateValidatorBuildTimeConfig;
```

Comment:
Oh man... I don't like this pattern at all, it should be disallowed really :). But there is now way I'm going to hold up the PR on a personal opinion :)

Reply:
Well, what do you suggest? Because it's how we always did things.

Note that I cannot use a producer because I want the ValidatorFactory to be produced at static init and there's no way to force that right now.

Reply:
I meant that ideally these arguments should be method arguments of the build step, not fields, which is just weird.

But it's not a problem obviously 😉

Reply:
An alternative could be to make `hibernateValidatorBuildTimeConfig ` as method parameter. Where it is used only.

Reply:
Ah yes, well, I chose consistency but sure I can change it.

Reply:
No need to change it now. Some other time, but it on a Christmas list 😉

Reply:
Done.